Tella/Scenes/Reports/View Model/ReportsViewModel.swift

```diff
@@ -15,12 +15,10 @@ class ReportsViewModel: ObservableObject {
     @Published var outboxedReports : [Report] = []
     @Published var submittedReports : [Report] = []
     @Published var selectedReport : Report?
-    @Published var selectedCell = Pages.draft
-    
-    var pageViewItems : [PageViewItem] {
-        [PageViewItem(title: LocalizableReport.draftTitle.localized, page: .draft, number: draftReports.count),
-         PageViewItem(title: LocalizableReport.outboxTitle.localized, page: .outbox, number: outboxedReports.count),
-         PageViewItem(title: LocalizableReport.submittedTitle.localized, page: .submitted, number: submittedReports.count)] }
+    @Published var selectedCell: Int = ReportPages.draft.rawValue
+    @Published var pageViewItems : [PageViewItem] = [PageViewItem(title: LocalizableReport.draftTitle.localized, page: ReportPages.draft.rawValue, number: "") ,
```

Comment:
I fixed that in the previous version because in the new iOS it doesn't work, if you can adapt the new `PageViewItem` with the previous fix :

> var pageViewItems : [PageViewItem] {
        [PageViewItem(title: LocalizableReport.draftTitle.localized, page: .draft, number: draftReports.count),
         PageViewItem(title: LocalizableReport.outboxTitle.localized, page: .outbox, number: outboxedReports.count),
         PageViewItem(title: LocalizableReport.submittedTitle.localized, page: .submitted, number: submittedReports.count)] }

Reply:
we change the implementation of pageView in this [commit](https://github.com/Horizontal-org/Tella-iOS/pull/74/commits/311ffd75fd0f51bc0706635e2608bad04c91269c#diff-e1db641c8e0123fd1916056289c2ba92aa72a91cbf588877bea522a2a9bf5b2b) to be able to re use it in uwazi and futures integrations. Why do you suggest to change it? in iOS 17.2.1 is working fine for me

Reply:
Could you check with the number of reports 
scenario: Create a draft -> the number of draft report will increase in my device doesn't change
just you need to adapt the new `PageViewItem` to the previous implementation 
I think it will be :

```
 var pageViewItems : [PageViewItem] {
    [PageViewItem(title: LocalizableReport.draftTitle.localized, page: ReportPages.draft.rawValue , number: "\(draftReports.count)"),
    PageViewItem(title: LocalizableReport.outboxTitle.localized, page: ReportPages.outbox.rawValue, number: "\(outboxedReports.count)"),
    PageViewItem(title: LocalizableReport.submittedTitle.localized, page: ReportPages.submitted.rawValue, number: "\(submittedReports.count)")] }
```

Reply:
got it! I just rollbacked to the previous implementation.